src/containers/HeatMap/heatMap.css

```diff
@@ -0,0 +1,145 @@
+.heat-map {
+  margin: 0 auto;
+  padding: 0 0 5rem 0;
+}
+
+.heat-map__header {
+  padding: 20px;
+  text-align: left;
+}
+
+.heat-map__logo {
+  height: 35px;
+}
+
+.heat-map__month {
+  padding: 0 1.5rem;
+  color: rgba(36, 36, 36, 0.9);
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 500;
+  text-align: center;
+}
+
+.heat-map__icon {
+  margin-right: 0.5rem;
+  color: rgba(36, 36, 36, 0.9);
+}
+
+.heat-map__body,
+.heat-map__journals {
+  padding: 0 1.2rem 1rem;
+}
+
+.heat-map__legend {
+  padding: 0 1rem 1.5rem;
+  display: flex;
+  justify-content: flex-end;
+  align-items: baseline;
+}
+
+.heat-map__box {
+  width: .8rem;
+  height: .7rem;
+  margin: 0 .3rem;
+}
+
+.heat-map__box--scale1 {
+  background-color: #feddd2;
+}
+.heat-map__box--scale2 {
+  background-color: #fec3b1;
+}
+.heat-map__box--scale3 {
+  background-color: #f2ac95;
+}
+.heat-map__box--scale4 {
+  background-color: #fc8d6b;
+}
+
+.heat-map__text {
+  text-align: center;
+  color: rgba(36, 36, 36, 0.6);
+  font-size: .8rem;
+  font-family: 'Open Sans', sans-serif;
+}
+
+.heat-map__empty {
+  display: flex;
+  padding: .5rem 1.5rem;
+}
+
+.heat-map__journals__message {
+  font-size: 1.2rem;
+  text-transform: capitalize;
+  color: rgba(36, 36, 36, 0.8);
+  font-family: Poppins, sans-serif;
+}
+
+/* The colors of the heat map */
+.react-calendar-heatmap .color-scale-1 {
+  fill: #feddd2;
+}
+.react-calendar-heatmap .color-scale-2 {
+  fill: #fec3b1;
+}
+.react-calendar-heatmap .color-scale-3 {
+  fill: #f2ac95;
+}
+.react-calendar-heatmap .color-scale-4 {
+  fill: #fc8d6b;
+}
+
```

Comment:
@yosefalnajjarofficial It might be helpful to move the `.heat-map__box--scale1` styles (lines 47-58) to be down here, so that they are right next to these color values.  Even though the selectors and the property that they set are different, the four color values are the same so if you put them right next to each other in the file, it's more likely that you would remember to change both sets of values, if you make color adjustments in the future.